src/buildercore/terraform.py

```diff
@@ -61,6 +61,16 @@ def render(context):
             }
         },
     }
+    if context['fastly']['gcslogging']:
+        gcslogging = context['fastly']['gcslogging']
+        # TODO: require FASTLY_GCS_EMAIL env variable
+        # TODO: require FASTLY_GCS_SECRET env variable
```

Comment:
no one wants to copy arbitrary credentials on a laptop to be able to run `update_infrastructure`, I think the recently revamped S3 module that manages EC2 SSH keys can be extended to store also these credentials (they are not keypairs, but similar) /cc @lsh-0

Reply:
are they files that are generated for us by Fastly?

Reply:
The credentials are generated by our Google Cloud console, and are needed for Fastly to write to Google Cloud Storage; means we need to push them into the Fastly Terraform template so that they are available to their servers. I think Terraform must have some standard way of handling credentials so I'll look for some standard pattern there first.